src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -291,6 +293,29 @@ export function createAnimatedComponent(
     _updateNativeEvents(
       prevProps: AnimatedComponentProps<InitialComponentProps>
     ) {
+      // If the event view tag changes, we need to completely re-mount all events
+      const computedEventTag = this._getEventViewTag();
```

Comment:
`computedEventTag` or `componentEventTag`? 🤔

Reply:
@piaskowyk computed - it is being calculated on the go and used for some calculations, but not yet applied